### ISSUE TITLE
my fkn eclipse wont start up anymore.... 

### ISSUE THREAD

**Issue by Ezoteric** [whichonespink44/Realistic-Terrain-Generation#163]
WHAT THE FUCK !!!!!!!!!!!!!!!!!!!!!!!

If i cant get it to work soon ill fuckin kill someone !!!!!!!!!!!

**Comment by whichonespink44** [comment]
What version do you use? Mars? I had some issues with Mars and had to go back to using Luna, which has been working a treat, as always.

Hope you get it sorted mate!

**Comment by Ezoteric** [comment]
well, i just hate when program just stop working when NOTHING changed....

I think its mars yes, but reinstalling it now hope it works cause i dont want to spend more time figuring it out haha

**Comment by Ezoteric** [comment]
you dont even know how long it took me to get eclipse working and graldew and finally knew how to compile and decompile :P

**Comment by Ezoteric** [comment]
AAAAAAAAAAAAARG, it quits without any error, it just shows that eclipse start up screen for not even a second :(

**Comment by Ezoteric** [comment]
when i try to start eclipsec in cmd it says 
Error occured during initialization of VM (java/lang/NoClassDefFoundError: java/lang/Object)

**Comment by Ezoteric** [comment]
fuck this shit. im done dont have time for this crap

**Comment by whichonespink44** [comment]
Does this sound like the problem you're having: https://geekycoder.wordpress.com/2009/07/08/java-tips-adventure-in-resolving-the-java-error-error-occurred-during-initialization-of-vm/

**Comment by whichonespink44** [comment]
TL;DR = if you've tried everything else, try delete java.exe from your C:\Windows\System32 directory.

**Comment by Ezoteric** [comment]
yeah i aint got java.exe in my win dir

matter of fact i checked 10 websites with different things :(

You dont see me complaining before i checked google :P

**Comment by Ezoteric** [comment]
was the first site i came on to and 

path=c:\devsys\java\lib\sdk16\bin
and voila, the problem goes away now ! does not solve it

i can run java from everywhere in CMD just eclipse does not work, and turst me NOTHING changed since yesterday, i quit eclipse went to bed now im home and it doesnt work anymore..... 

I really get pissed off cause of shit like this GRRRRRRRRRRRRRRRRRRR

its good i smoke weed, so i can calm the f down

**Comment by Ezoteric** [comment]
ended up reistalling every thing, java eclipse and works now, but if this gonna happen everyday praise the lord i wont ddos them !!!

**Comment by Ezoteric** [comment]
Besides that, if we get islands from MCs gen layer and would be able to add the volcano on those islands it doesn't matter if it's still called ocean. Cause I think you can only name biomes right ?

Also gen layers add shores so that's a good a thing but I think it will be some work to get that in as well.  I think I'll. Check how bop did it cause that code seems a lol easier then cc code

**Comment by Ezoteric** [comment]
Lol wrong one haha damnz phone
